Automattic/jetpack
# WhatsApp Button: Fix misalignment on mobile devices

Fixes https://github.com/Automattic/jetpack/issues/16404

#### Changes proposed in this Pull Request:

Fix the alignment of the WhatsApp button when on mobile devices by setting a fixed font size.

**Before**
Text is misaligned

![86570263-ec99d100-bf6f-11ea-90d3-e60d2f8430ca](https://user-images.githubusercontent.com/1464705/98740200-8a0e7280-235f-11eb-9a27-834674ddbb87.png)

**After**
Text is now aligned, and font size matches the desktop

<img width="437" alt="Screen Shot 2020-11-10 at 2 18 27 PM" src="https://user-images.githubusercontent.com/1464705/98740272-9b577f00-235f-11eb-98eb-d0c1cda890db.png">

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:






* Check out this PR
* Create a new post and add a WhatsApp button
* Load the post and enter responsive mode.
* Make sure the canvas is less than 560px wide
* Confirm that the WhatsApp button remains aligned.

#### Proposed changelog entry for your changes:

* WhatsApp button: Fixed issue with text alignment on mobile devices.
